src/io/rainrobot/maxdispachreq/LoadBalanceService.java

```diff
@@ -1,79 +1,42 @@
 package io.rainrobot.maxdispachreq;
 
-
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-
 public class LoadBalanceService {
 
     private final int MAX_REQUESTS;
-    private int maxAvailability;
-    private Map<Integer, Set<serverAddress>> avelabiletyServerMap;
-    private io.rainrobot.maxdispachreq.serverManeger serverManeger;
+    private AvailabilityService avelabiletyService;
```

Comment:
I think this service is not well define, maybe you should define in the comments of the service what its role.